packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapGroundOverlayController.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+
+#import "messages.g.h"
+
+// Defines ground overlay controllable by Flutter.
+@interface FLTGoogleMapGroundOverlayController : NSObject
```

Comment:
This and all the methos below need to be documented.
See heatmaps controller as a reference.